app/controllers/close_contacts_controller.rb

```diff
@@ -2,8 +2,33 @@
 
 # CloseContactsController: close contacts
 class CloseContactsController < ApplicationController
+  include CloseContactQueryHelper
+
   before_action :authenticate_user!
 
+  def index
+    redirect_to(root_url) && return unless current_user&.can_view_patient_close_contacts?
+
+    # Validate params and handle errors if invalid
+    begin
+      data = validate_close_contact_query(params)
```

Comment:
can we rename `data` to `query` (same for all the other controllers this is applicable to)

Reply:
sure thing!